components/common/header.tsx

```diff
@@ -0,0 +1,87 @@
+import styled from "@emotion/styled";
+import { useState } from "react";
+import { color, text } from "@/styles/theme";
+import Image from "next/image";
+import ProfileImage from "@/components/common/profileImage";
+import LogoutTooltip from "@/components/common/logoutTooltip";
+
+const Header: React.FC = () => {
+  const [show, setShow] = useState(false);
+
+  const toggle = () => {
+    setShow(!show);
+  };
+
+  return (
+    <StyledHeader>
+      <Logo>
+        <Image src="/icon/logo.svg" alt="logo" width={252} height={48} />
+      </Logo>
+      <NavigatorWrapper>
+        <Navigator>피드</Navigator>
+        <Navigator>머구리 찾기</Navigator>
+        <Navigator>마이페이지</Navigator>
+      </NavigatorWrapper>
+      <UserWrapper>
+        <AlarmButton>
+          <Image src="/icon/bell.svg" alt="alarm" width={30} height={30} />
+        </AlarmButton>
+        <UserImg onClick={toggle}>
+          <ProfileImage size="sm" />
+        </UserImg>
+        <UserName onClick={toggle}>username</UserName>
+        <LogoutTooltip
+          onMouseLeave={toggle}
+          style={{ display: show ? "block" : "none" }}
+        />
+      </UserWrapper>
+    </StyledHeader>
+  );
+};
+
+export default Header;
+
+const StyledHeader = styled.header`
+  height: 62px;
+  padding: 0 20px;
+  display: flex;
+  justify-content: space-between;
+  border-bottom: 2px solid ${color.$gray400};
+`;
+
+const Logo = styled.div`
+  cursor: pointer;
+`;
+
+const NavigatorWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+`;
+
+const Navigator = styled.div`
+  margin: auto 20px;
```

Comment:
```suggestion
const NavigatorWrapper = styled.div`
  display: flex;
  justify-content: space-between;
  gap: 34px;
`;
const Navigator = styled.div`
```

`display: flex`인 상태라면 `gap` 사용으로 쉽게 flex-item 사이 여백을 설정할 수 있습니다!